coralogix-eventbridge-policy/template.yaml

```diff
@@ -6,6 +6,11 @@ Parameters:
     Description: The ARN corresponding to the Event Bus that will receive events via the PutEvents method.
     AllowedPattern: '^arn:aws[\w-]*:events:[a-z]{2}-[a-z]+-[\w-]+:[0-9]{12}:event-bus\/[\.\-_A-Za-z0-9]+$'
     MaxLength: 2048
+  RoleName:
+    Type: String
+    Description: Don't change it! It needs to match the one that was input on the Coralogix form. Corresponds to the name corresponding to the AWS IAM role that will be created.
```

Comment:
> Corresponds to the name corresponding

Sounds a bit weird. I'd suggest to rephrase to:
> Corresponding to the AWS IAM role name that will be created

Reply:
Good catch. I changed to `Corresponds to the name of the AWS IAM role that will be created.` as that seems to be the most grammatically correct.